### DISCUSSION TITLE
the NVDA community and accessibility allies are now also on Discord

### DISCUSSION THREAD

**Discussion by Adriani90** [nvaccess/nvda#16026] [Show and tell]
### Background
For many years we tried to connect the community via events on Teamtalk, but this had a lot of challenges:
* Audio issues (i.e. poor audio quality)
* Limited features for administration / moderation
* Difficulties in connecting local country specific communities and the international community due to server limitations
* low popularity of Teamtalk users in general.

Also non disabled people who use NVDA in other circumstances (i.e. for testing software accessibility) were not really connected to the broader NVDA community unless they registered for the mailing lists.
The result of this development are tons of mailing lists, facebook groups and other channels with the same purpose of building a NVDA community, especially country specific mailing lists or whats app groups were totally disconnected from the international community due to language barreers.

### Current solution
While the current channels are definitely worthy and they should exist in parallel, Discord servers provide a lot of advantages.
In case you use NVDA on a regular basis or for testing the accessibility of your software products or websites, we have now a nitro level 1 Discord Plan for the NVDA community. The purpose of this server is continuous interactions within the NVDA community and we are also having a stage room where we can host accessibility webinars, Testing sessions, WCAG webinars with focus on NVDA functionality etc.  Every community event such as NVDACon will take place there. The server contains a general accessibility chat room where you can raise your questions, forum channels separated for users, translators and developers, a tutorial channel and audio channels where you can speak with other people from the NVDA community and accessibility allies in general. On this server we connect for the first time the international and the local communities in a much easier way. If you want to join, please follow this link: 
https://discord.gg/ynhxWvbs6a
More details about the server you can find on the general server information channel there.
Developers and active github contributors have a dedicated area where only they have the special role to post new topics, anyone else can read only. i designed it like that to avoid noisy chats in the development area.

### Advantages of the server design
* The roles management is very comprehensive and can be done per category, so people have their own communities in local languages and are still not far away from the english international community.
* The bot environment for Discord is very powerful. We can connect the current announcement channels such as NV Access news website, and the addons mailing list with the Discord announcement channel, so people who don't want to come to Discord can still send announcements for new addons etc. to the usual mailing list, and NV Access can just post news on their website which will be automatically sent to the announcement channel in Discord.
* Automatic translation can be performed via a bot solution, and thus will give people who don't speak english a way to express themselves in the international community as well.
* The server includes also links to the support page for NV Access and to the NV Access shop, and these do not disappear from the server.
* The Discord application as such is very accessible for blind and visually impaired users, and the company is very responsive for feedback.
* There are already a lot of companies actively using Discord, and also many developers use it as an agile work platform. So connecting them with the NVDA community is now much easier.

### Drawbacks
* Currently the captcha solving requirement is not accessible for blind and visually impaired people, so the registration process is challenging without sighted assistance. But we are in discussions with the Discord company to change that asap.

### Current server status
* As of this post, around 136 people joined the server. I will take some time until people get used to it but communication is slowly starting within the channels
* We have a dedicated area for the India, Netherlands and France communities so far.

So everyone is welcome to contribute on the server.

**Comment by lpintes** [comment]
Hello,
Unfortunately I was unable to sign-up. The problem was not with HCaptcha, but with "Enter the date of birth". Although it looked okay, the same dialog popped up again and again after I clicked next.
But interestingly the process looked to be very accessible.

**Comment by josephsl** [comment]
Hi,

I doubt it - I assume Discord. Besides, promoting a YouTube channel when we are talking about Discord is off-topic.

**Comment by Adriani90** [comment]
@lpintes I think on the mobile phone is more accessible than on the desktop app. Or you could also try it on the website without any of the discord apps at least for the registering process.

**Comment by lpintes** [comment]
To try this, I used the provided link.

Dňa 10. 1. 2024 o 18:54 Adriani90 napísal(a):

**Comment by Adriani90** [comment]
Ah ok, you could open that link in the discord app itself, so you probably did it in the browser right?

 

 

Von: Ľuboš Pinteš ***@***.***> 
Gesendet: Mittwoch, 10. Januar 2024 19:57
An: nvaccess/nvda ***@***.***>
Cc: Adriani90 ***@***.***>; Author ***@***.***>
Betreff: Re: [nvaccess/nvda] the NVDA community and accessibility allies are now also on Discord (Discussion #16026)

 

To try this, I used the provided link.

Dňa 10. 1. 2024 o 18:54 Adriani90 napísal(a):

**Comment by lpintes** [comment]
Yes I clicked to the link in e-mail.


Dňa 10. 1. 2024 o 20:22 Adriani90 napísal(a):

**Comment by florianionascu7** [comment]
Hello! I'm excited to be part of this Discord server. Adriani, it's a great job!
